crates/holochain_state/src/source_chain.rs

```diff
@@ -850,8 +850,13 @@ pub fn put_raw(
         // StoreEntry ops with private entries are never gossiped or published
         // so we don't need to integrate them.
         // TODO: Can anything every depend on a private store entry op? I don't think so.
+        set_validation_status(
+            txn,
+            op_hash.clone(),
+            holochain_zome_types::ValidationStatus::Valid,
+        )?;
```

Comment:
I know that put_raw is only used to create the genesis entries, but I think it would be really confusing if a raw put automatically set everything as valid, if it were used anywhere else (in a test, etc). I think you should at least pass in a bool flag where if true, set the validation like so.